internal/rollup/rollup_bundle.bzl

```diff
@@ -611,10 +611,19 @@ ROLLUP_ATTRS = {
         It is sufficient to load one of these SystemJS boilerplate/entry point
         files as a script in your HTML to load your application""",
     ),
+    "auto_additional_entry_points": attr.bool(
```

Comment:
decided to rename it "enable_code_splitting" since this is orthogonal to the number of entry points